### ISSUE TITLE
Question about data used for three tasks

### ISSUE THREAD

**Issue by JaniceXiong** [allenai/scicite#12]
Hi! I read the original paper and have a question about data used for three tasks.
  In paper(2.2 Training), the loss function is like picture below, and (x,y) from three dataset respectively. I suppose x is the same citation, but get y from difference resourses (the section title, the worthiness, the intent respect to the same x)
![image](https://user-images.githubusercontent.com/43976565/100747031-77e29a00-341c-11eb-9633-7b38c52dd5a1.png)
  But when I look through the code, I find there is no connection between the generated data batch, batch_aux, batch_aux2. In other words, the x is totally different. 
  If so, why not pretrain the shared part through two auxiliary tasks directly and fine-tuning the classifier for the main task? And if pretrained, the auxiliary task data is more larger compared to the multitask framework.
  Thanks for your wonderful work and Hope you have a nice day :D
